v3/integrations/nrpkgerrors/nrkpgerrors_test.go

```diff
@@ -31,10 +31,22 @@ func gamma(e error) error { return errors.WithStack(e) }
 
 func theta(e error) error { return errors.WithMessage(e, "theta") }
 
+func basicNRError(e error) newrelic.Error { return newrelic.Error{Message: e.Error()} }
+func withAttributes(e error) newrelic.Error {
+	return newrelic.Error{
+		Message: e.Error(),
+		Attributes: map[string]interface{}{
+			"testAttribute": 1,
+			"foo":           2,
+		},
+	}
+}
+
 func TestWrappedStackTrace(t *testing.T) {
 	testcases := []struct {
-		Error          error
-		ExpectTopFrame string
+		Error            error
+		ExpectTopFrame   string
+		ExpectAttributes map[string]interface{}
 	}{
 		{Error: basicError{}, ExpectTopFrame: ""},
 		{Error: alpha(basicError{}), ExpectTopFrame: "alpha"},
```

Comment:
Odd indentation with the next two lines -- are these line feeds or something? GitHub won't let me comment directly on them.